client/src/Services/auth-services.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
```

Comment:
Did @skassam21 want to call this folder `utils` ? That's what I remember but i can't find it mentioned anywhere. Also, I don't think directories should be capitalized (I'm traumatized from the last capitalization issue!!!)?